www/_posts/2024/2024-01-22-outstanding-award.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post.html
+title: "Announcing: The Outstanding PyLady Award"
+tags: [PyLadies, Global, Community, Outreach]
+author: Lynn Root
+author_link: https://mastodon.online/@roguelynn
+---
+
+Introducing: the Outstanding PyLady Award!
+
+**TL;DR:** Know of a PyLady that should be recognized for all their work for Python and the community?
+[Nominate them (or yourself)][nomination form]!
+
+The PyLadies community is made up of many remarkable members and volunteers that make significant 
+contributions to bettering the Python community. The Global Council's [Outstanding PyLady award][award docs] is 
+meant to elevate diverse Pythonistas doing amazing things, to highlight their achievements, and 
+to thank them for all the work they've contributed.
+
+The award is open to women and non-binary PyLadies. Any Python community member may nominate a 
+PyLady at any time (including self-nominations! go on – brag about yourself!). 
+**Nominations are accepted via [this Google form][nomination form].**
+
+Nominations are accepted year-round, and award winners will be recognized at the nearest upcoming PyCon US.
+**For recognition at PyCon US 2024, nominations must be in by February 4th 2024 [AOE].**
+
+For more information about the process, please have a look at our [documentation][award docs].
+
+_The Global Council wants to give a huge thank you to [Cheuk Ting Ho] for proposing the idea of 
+recognizing & rewarding individual PyLadies who give a lot of time and effort to make Python and 
+its community better!_
+
+[nomination form]: https://docs.google.com/forms/d/e/1FAIpQLSccnm4kYFez8SPWJD7ZfaAojHZE0qoLF_MLarGzzzc3pcY-xA/viewform
+[award docs]: https://kit.pyladies.com/global/award.html
```

Comment:
This URL is 404

Reply:
Clicking it from GH does 404, but pasting it into a browser renders the form.

Reply:
I was referring to the award docs url:
https://kit.pyladies.com/global/award.html
is 404

Reply:
I think the URL slugs changed with the change to Furo cc/ @econchick 
The published URL is now https://kit.pyladies.com/en/latest/global/award.html
we might need to do some redirection magic as it seems all the kit.pyladies.com/global URLs will be broken 😱